utils.py

```diff
@@ -311,6 +311,22 @@ def get_current_time_in_millisecs():
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
+def get_current_date_as_string():
```

Comment:
I just noticed that this refers to feconf.DASHBOARD_STATS_DATETIME_STRING_FORMAT, which is dashboard-stats-based, but that this function (and the following one) are in utils.py, which is a general file for the entire codebase, and are named generically.

It's unlikely that we'll use the same date format everywhere, so I suggest either renaming this to get_dashboard_stats_string_for_current_date() and get_date_from_dashboard_stats_string(), or moving them to the user_services domain file. In addition, add a comment next to the constant along the lines of "NOTE TO DEVELOPERS: this format should not be changed, since it is used in the existing storage models for UserStatsModel" -- this will hopefully prevent people from accidentally changing the format string and thereby messing up the existing stats models.